### DISCUSSION TITLE
[FEATURE] GIF viewing support

### DISCUSSION THREAD

**Discussion by samip5** [immich-app/immich#1596] [Feature Request]
**Describe the bug**
The uploaded gif doesn't play like one would expect.

**To Reproduce**
1. Go to giphy.com and download a gif.
2. Upload the gif to immich
3. Refresh page
4. See that it doesn't move.

**Expected behavior**
I would expect the gif to actually play like a gif, aka be moving.

**Screenshots**
![image](https://user-images.githubusercontent.com/1703002/215221760-2197751f-2183-43f8-b106-f9734ed873c5.png)

**System**
- Server Version: 1.43.0

**Additional context**
I used this as my test.gif: https://giphy.com/gifs/moodman-YRtLgsajXrz1FNJ6oy

**Comment by JVT038** [comment]
I have the same problem

**Comment by alextran1502** [comment]
This is not supported yet

**Comment by samip5** [comment]
This has been resolved since https://github.com/immich-app/immich/pull/2022
